extensions/hibernate-reactive/deployment/src/test/java/io/quarkus/hibernate/reactive/SchemaUpdateTest.java

```diff
@@ -28,12 +27,15 @@
     @Inject
     Mutiny.SessionFactory sessionFactory;
 
+    @RunOnVertxContext
     @Test
-    public void integerIdentifierWithStageAPI() {
-        final Uni<List<Object>> result = sessionFactory.withSession(s -> s
-                .createQuery("from Hero h where h.name = :name").setParameter("name", "Galadriel").getResultList());
-        final List<Object> list = result.await().indefinitely();
-        assertThat(list).isEmpty();
+    public void integerIdentifierWithStageAPI(UniAsserter asserter) {
```

Comment:
Not sure I like that, but I understand why it goes in this direction. 

The problem is that the API is totally different from the UniAssertSubscriber from Mutiny.

Reply:
Yes, it is... But I am not sure what we can do about it...